luaui/Widgets/cmd_bar_hotkeys.lua

```diff
@@ -33,37 +25,82 @@ local function reloadWidgetsBindings()
 	end
 end
 
-local function loadBindings()
-	for _, v in ipairs(makeBindsTable()) do
+local function makeBindsTable(file)
```

Comment:
Just a reminder about 1 or a few months after this is merged the codepath for lua format migration can be removed.